packages/dappmanager/test/integration/hostScripts.test.int.ts

```diff
@@ -7,21 +7,21 @@ const hostScriptsPath = process.cwd() + "/hostScripts";
 describe("Host scripts", () => {
   it("Should fetch docker engine update requirements", async () => {
     const dockerEngineUpdateRequirements = await shell(
-      `bash ${hostScriptsPath}/docker_engine_update.sh --print-host-info`
+      `sudo bash ${hostScriptsPath}/docker_engine_update.sh --print-host-info`
```

Comment:
Why do you need sudo now?

Reply:
because it creates a log file at a reserved path: /usr/src/dappnode/logs

Reply:
Please find an alternative way to do that. Using sudo just for logs is not good

Reply:
https://github.com/dappnode/DNP_DAPPMANAGER/pull/748#discussion_r625614512